src/diff/index.js

```diff
@@ -110,6 +109,11 @@ export function diff(dom, parentDom, newVNode, oldVNode, context, isSvg, excessD
 					break outer;
 				}
 
+				if (newType._id==='_P' && c.props.value!==newVNode.props.value) {
+					const v = newVNode.props.value;
+					newVNode._component.subs.map(sub => v!==sub.context && (sub.context = v, enqueueRender(sub)));
```

Comment:
Something I was wondering, can these two ever be equal? I don't really think so.

Reply:
I removed it and nothing in test results changed. 

We are comparing v which is a value to a Context Object so it always evaluates to true. This saves 9Bytes